docs/portal.properties-Reference.md

```diff
@@ -68,6 +68,18 @@ The default view in OncoPrint ("patient" or "sample") can be set with the follow
 oncoprint.defaultview=sample
 ```
 
+## Custom annotation of driver and passenger mutations
+cBioPortal allows for two different formats of custom annotation of driver and passenger mutations in the MAF files (see [file format documentation](File-Formats.md#extending-the-maf-format)). To enable the view of those annotations (just one or all both), you must specify a name for the label(s) that you want to enable (those labels appear in the oncoprint "Mutation color" menu):
+```
+oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
+oncoprint.custom_driver_annotation.tiers.menu_label=Custom tiers
+```
+
+By default, when at least one of the custom annotations is enabled, custom annotations are automatically selected, instead of OncoKB and Hotspots. If you wish to change this default behavior and have OncoKB and Hotspots as the default selected annotations, set this parameter to false:
```

Comment:
Yes, could you rephrase it to ".... one of the custom annotations is enabled in portal.properties and the required columns are present in MAF files, custom annotations are......."